content/docs/fairing/azure/az-kubeflow-notebook.md

```diff
@@ -0,0 +1,186 @@
+     `client_id` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_CLIENT_SECRET:** The Service Principal secret. You can get the
+     `client_secret` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_TENANT_ID:** The Azure Tenant ID of your account. You can get the
+     Tenant ID from the `tenantId` field in the output of `az account show`.
+   * **AZ_SUBSCRIPTION:** The Azure Subscription ID of your account. You can
+     get the Subscription ID from the `id` field in the output of `az account
+     show`.
+   * **TARGET_NAMESPACE:** Specify the namespace that your Notebook Server is
+     in. For example, this guide recommends using `kubeflow-anonymous`.
+   * **ACR_NAME:** The name of an ACR that the Service Principal can access.
+
+1. Use the following command to create a secret that Kubeflow can use to access
+   Azure APIs.
+
+   ```bash
+   kubectl create secret generic -n ${TARGET_NAMESPACE} azcreds \
+   --from-literal=AZ_CLIENT_ID=${AZ_CLIENT_ID} \
+   --from-literal=AZ_CLIENT_SECRET=${AZ_CLIENT_SECRET} \
+   --from-literal=AZ_TENANT_ID=${AZ_TENANT_ID} \
+   --from-literal=AZ_SUBSCRIPTION_ID=${AZ_SUBSCRIPTION_ID}
+   ```
+
+1. Use the following command to create a secret that Kubeflow can use to access
+   ACR.
+
+   ```bash
+   kubectl create secret docker-registry -n ${TARGET_NAMESPACE} acrcreds \
+   --docker-server=${ACR_NAME}.azurecr.io \
+   --docker-username=${AZ_CLIENT_ID} \
+   --docker-password=${AZ_CLIENT_SECRET}
+   kubectl patch serviceaccount default-editor -n ${TARGET_NAMESPACE} \
+   -p "{\"imagePullSecrets\": [{\"name\": \"acrcreds\"}]}"
+   ```
+
+## Creating a Notebook Server in Kubeflow
+
+To create a Notebook Server, use your Web browser to access the Kubeflow
+Central Dashboard and select the `Notebook Servers` panel from the menu.
+
+First, select the target namespace in which you want to host the server. In the
+default Kubeflow installation, there should be a namespace `kubeflow-anonymous`
+available in the namespace drop-down menu.
+
+After the target namespace is selected, click `NEW SERVER` and fill in the
+mandatory fields. The fields with default values can all be left as they
+are and do not have to be modified to run the example notebook.
+
+After launching the server, wait for the `CONNECT` button to appear and click
+`CONNECT` to launch your Notebook Server. It may take up to a minute for the
+server to be ready for connections.
+
+## Cloning the example notebook
+
+Clone the Kubeflow Fairing repository to download the files used in this example. 
+
+1. Connect to your notebook server, then click the new terminal option
+   like in the screenshot below:
+
+   <img src="/docs/images/azure-notebook-new-terminal.png"
+        alt="Creating new terminal after connecting to notebook server"
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Run the following command to clone the Kubeflow Fairing project:
```

Comment:
List formatting is broken. Probably the same cause as mentioned in my previous comment.